pkg/gadgets/trace/tcpconnect/tracer/tracer.go

```diff
@@ -123,19 +132,31 @@ func (t *Tracer) install() error {
 		return fmt.Errorf("error attaching program: %w", err)
 	}
 
-	t.v4ExitLink, err = link.Kretprobe("tcp_v4_connect", t.objs.IgTcpcV4CoX, nil)
-	if err != nil {
-		return fmt.Errorf("error attaching program: %w", err)
-	}
-
 	t.v6EnterLink, err = link.Kprobe("tcp_v6_connect", t.objs.IgTcpcV6CoE, nil)
 	if err != nil {
 		return fmt.Errorf("error attaching program: %w", err)
 	}
 
-	t.v6ExitLink, err = link.Kretprobe("tcp_v6_connect", t.objs.IgTcpcV6CoX, nil)
-	if err != nil {
-		return fmt.Errorf("error attaching program: %w", err)
+	if !t.config.CalculateLatency {
```

Comment:
Suggestion: add a comment before:
- SEC("kretprobe/tcp_v4_connect")
- SEC("kretprobe/tcp_v6_connect")
- SEC("tracepoint/tcp/tcp_destroy_sock")
- SEC("kprobe/tcp_rcv_state_process")

to say something like:
// This kretprobe/tracepoint/kretprobe is only attached if calculate_latency is true|false